fastlane/Fastfile

```diff
@@ -549,17 +548,50 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       download_path: "./fastlane/metadata"
     )
 
-    # Create the marketing and privacy URL files
-    metadata_locales.each { |locale|
-      marketing_path = Dir.pwd + "/metadata/#{locale}/marketing_url.txt"
-      write_file(marketing_path, 'https://woocommerce.com/mobile/')
-      
-      privacy_path = Dir.pwd + "/metadata/#{locale}/privacy_url.txt"
-      write_file(privacy_path, "https://automattic.com/privacy/")
+    # Ensure that localizations are present in the right places (some locale codes aren't correct)
+    {
+      "ar" => "ar-SA",
+      "de" => "de-DE",
+      "es" => "es-ES",
+      "fr" => "fr-FR",
+      "nl" => "nl-NL",
+      "zh-cn" => "zh-Hans",
+      "zh-tw" => "zh-Hant"
+    }.each do |source, destination|
+      source_dir = File.join(Dir.pwd, "metadata", source)
+      Dir.children(source_dir).each do |file| 
+        source_file = File.join(source_dir, file)
+        destination_file = File.join(Dir.pwd, "metadata", destination, file)
+        FileUtils.mv(source_file, destination_file)
+      end
+      FileUtils.remove_dir(source_dir)
+    end
 
-      support_path = Dir.pwd + "/metadata/#{locale}/support_url.txt"
-      write_file(support_path, "https://docs.woocommerce.com/")
-    }
+    # Copy source files into `en-US`
+    enUS_path = File.join(Dir.pwd, 'metadata', 'en-US')
+
+    FileUtils.mkdir_p(enUS_path)
+
+    [
+      "promo_screenshot_1.txt",
+      "promo_screenshot_2.txt",
+      "promo_screenshot_3.txt",
+      "promo_screenshot_4.txt",
+      "promo_screenshot_5.txt",
+    ].each do |filename|
+      source = File.join(Dir.pwd, 'appstoreres', 'metadata', 'source', filename)
+      destination = File.join(enUS_path, filename)
+      FileUtils.cp(source, destination)
```

Comment:
@jkmassel Actually now that I'm trying to apply a similar PR to other repos, I'm wondering what was the point of this, what's the need of copying the txt from `fastlane/appstoreres/metadata/source` to `fastlane/metadata/en_US`? (Or was the original intention to copy them to `fastlane/appstoreres/metadata/en_US` instead?)